python-sdk/tests/benchmark/datasets.md

```diff
@@ -186,7 +186,9 @@ Subset of the Github git repository records
 
 ### Stack Overflow
 Subset of the archives of Stack Overflow:
+<!-- markdown-link-check-disable -->
 * Source: https://archive.org/download/stackexchange (superuser.com.7z)
```

Comment:
@utkarsharma2 this link works for me. https://archive.org/download/stackexchange

And there is also:
> [superuser.com.7z](https://archive.org/download/stackexchange/superuser.com.7z) ([View Contents](https://archive.org/download/stackexchange/superuser.com.7z/))

Reply:
Damn, this was not working earlier - https://github.com/astronomer/astro-sdk/actions/runs/3931016955/jobs/6721731345